src/Microsoft.Azure.WebJobs.Host/Queues/QueueProcessor.cs

```diff
@@ -211,13 +213,9 @@ public QueueProcessor(QueueProcessorFactoryContext context)
         /// Called to raise the MessageAddedToPoisonQueue event
         /// </summary>
         /// <param name="e">The event arguments</param>
-        protected internal virtual void OnMessageAddedToPoisonQueue(EventArgs e)
+        protected internal virtual void OnMessageAddedToPoisonQueue(PoisonMessageEventArgs e)
```

Comment:
Yes, customers set up queue trigger functions to process the poison queue. QueueProcessor is a lower level advanced thing, and if you see how it is used, you'll see that other aspects of our infra need to know when a poison message is added.